src/Azure/Orleans.Streaming.EventHubs/Providers/Streams/EventHub/EventHubCheckpointerOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Orleans.Configuration
+{
+    public class EventHubCheckpointerOptions
+    {
+        /// <summary>
+        /// Azure table storage connections string.
+        /// </summary>
+        [RedactConnectionString]
+        public string CheckpointConnectionString { get; set; }
```

Comment:
Since these are in their own options now (a good thing) we should remove the "Checkpoint" prefix from all the settings.  Those were only necessary for specificity in a large flat config object.